src/Quartz.Extensions.DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -20,6 +20,17 @@ public static class ServiceCollectionExtensions
         public static IServiceCollection AddQuartz(
             this IServiceCollection services,
             Action<IServiceCollectionQuartzConfigurator>? configure = null)
+        {
+            return AddQuartz(services, new NameValueCollection(), configure);
+        }
+
+        /// <summary>
+        /// Configures Quartz services to underlying service collection. This API maybe change!
+        /// </summary>
+        public static IServiceCollection AddQuartz(
+            this IServiceCollection services,
+            NameValueCollection properties,
```

Comment:
Using `NameValueCollection` in `IServiceCollection` is not very .NET Core/`Microsoft.Extensions.Options`-ish. If you're not against introducing breaking changes (I see a few ones are already introduced in other parts), maybe you could move this thing to a dedicated method on `IServiceCollectionQuartzConfigurator`? Something like `options.ImportProperties(new NameValueCollection())` that would call `services.Configure()` under the hood?

Reply:
Sounds reasonable, but I think that requires some changes... I'd rather leave that to later time as there's some scheduling (grin) pressure here to get a release out 😉 

For Quartz it's idiomatic to use `NameValueCollection` but that's a fair point that it's not something you'd see on public API.